README.md

```diff
@@ -8,13 +8,13 @@ An HTTP service for the controlling of search API
 
 Run new search in the UI:
 * run the [web journey](https://github.com/ONSdigital/dp/blob/main/guides/INSTALLING.md#web-journey)
-* set `SearchRoutesEnabled` to `true` and `SearchABTestPercentage` to `100` in the config of the [dp-frontend-router](https://github.com/ONSdigital/dp-frontend-router)
 
```

Comment:
I believe this [config variable and associated code] hasn't been removed yet... (https://github.com/ONSdigital/dp-frontend-router/search?q=SearchRoutesEnabled)